### PR TITLE
Addition of initial OpenAPI specification

### PR DESCRIPTION
**Pull request by dat-adi**
Set up the initial OpenAPI specification.
Sorted the required components into their respective directories.

Added,
- [x] Notifications
- [x] Issues
- [x] Repositories

### CODE REVIEW

File: docs/api/v1/components/responses/repo_link.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+type: object
```
Comment (realaravinth) [inline docs/api/v1/components/responses/repo_link.yaml:1]: duplicate of [`docs/api/v1/components/schema/repo_link.yaml`](https://github.com/forgefedv2/interface/blob/2366e3efb60ae8b8ef860f263cf267b1ec35354d/docs/api/v1/components/schema/repo_link.yaml)

**Review by realaravinth — CHANGES_REQUESTED** [review]
There seems to be a duplicate object, please get rid of that.

Also, can you setup build and test workflow like we did with Northstar?

File: docs/api/v1/components/responses/repo_link.yaml (hunk 1 above)
Comment (dat-adi) [reply docs/api/v1/components/responses/repo_link.yaml:1]: Noted, will be removing the duplicate object.

**Comment by dat-adi** [comment]
Setting up the build and test workflow may be a bit delayed, but, I'd be glad to work on that as well, no issues.
@realaravinth

**Comment by realaravinth** [comment]
We'll merge it after that stuff is setup

**Comment by dat-adi** [comment]
Corrected errors, and build and test configuration is set up and ready to go.

**Review by realaravinth — APPROVED** [review]

**Pull request merged**